handbook/sales/README.md

```diff
@@ -28,6 +29,13 @@ Rituals (out-dated 2023-10-19)
 ## Responsibilities
 The Sales department is directly responsible for attaining the revenue goals of Fleet and helping to deliver upon our customers' objectives.
 
+### Sales process
```

Comment:
@alexmitchelliii would you consider the first part strategy? Maybe it takes the place of "Cold outbound strategy".

Reply:
Not sure if I just accepted your suggestions, @Sampfluger88 . But that is what I was trying to do. I'm good splitting it into two PRs.